icu4c/source/common/brkiter.cpp

```diff
@@ -430,16 +431,24 @@ BreakIterator::makeInstance(const Locale& loc, int32_t kind, UErrorCode& status)
         {
             UTRACE_ENTRY(UTRACE_UBRK_CREATE_LINE);
             uprv_strcpy(lbType, "line");
-            char lbKeyValue[kKeyValueLenMax] = {0};
             UErrorCode kvStatus = U_ZERO_ERROR;
-            int32_t kLen = loc.getKeywordValue("lb", lbKeyValue, kKeyValueLenMax, kvStatus);
-            if (U_SUCCESS(kvStatus) && kLen > 0 && (uprv_strcmp(lbKeyValue,"strict")==0 || uprv_strcmp(lbKeyValue,"normal")==0 || uprv_strcmp(lbKeyValue,"loose")==0)) {
+            CharString value;
+            CharStringByteSink valueSink(&value);
+            loc.getKeywordValue("lb", valueSink, kvStatus);
+            if (U_SUCCESS(kvStatus) && (value == "strict" || value == "normal" || value == "loose")) {
                 uprv_strcat(lbType, "_");
-                uprv_strcat(lbType, lbKeyValue);
+                uprv_strcat(lbType, value.data());
+            } else {
```

Comment:
I looked yesterday and wasn't sure if they are orthogonal.

I found that I asked this in [August](https://docs.google.com/document/d/1il9KJH3MeSkghBZRD5XriGXk9gfxN3yrVXmRiiUR1q4/edit#heading=h.qqfnsikbpkct) and left it as a question in the to-do's: “Decide whether this is orthogonal to CSS line break strict/loose/... (This would be more complicated than if it could be just another value in the same namespace.)”

We had another meeting in November, and I think we discussed this again. Can someone please point me to the meeting minutes for that?

Reply:
So the [minutes for our 2021-nov-17 meeting](https://docs.google.com/document/d/11yJUWedBIpmq-YNSqqDfgUxcREmlvV0NskYganXkQHA/edit#heading=h.6194od71cpe2) are in the ICU agenda & minutes doc. (Eventually they will move into an archive copy.)

I see that we decided that the new option fits into the -lw key's namespace, but apparently we didn't discuss -lb vs. -lw.

I am ok with not blocking this PR on this question, but we need to define the relationship between the -lb and -lw keys, for both the CLDR spec and the ICU code.